config/schemas/config.yaml

```diff
@@ -127,6 +370,707 @@ properties:
           - containerd
           - docker
     additionalProperties: false
+  clusterApi:
+    additionalProperties: false
+    properties:
+      clusters:
+        items:
+          type: string
+        type: array
+      enabled:
+        default: false
+        type: boolean
+      monitoring:
+        additionalProperties: false
+        properties:
+          enabled:
+            default: false
+            type: boolean
+        type: object
+    type: object
+  user:
+    additionalProperties:
+      $ref: '#/$defs/component'
```

Comment:
Should user have this component? I can not at least see it having them. :thinking:

Reply:
I wasn't sure what else were under `.user`. Replacing it with explicit `enabled`-only objects for `sealedSecrets`, `mongodb`, `fluxv2`, and `kafka`.